modules/community-algorithms/index.adoc

```diff
@@ -0,0 +1,23 @@
+= Community Algorithms
+:description: Overview of community algorithms.
+
+Community algorithms group together vertices or edges that satisfy some rule for being connected to one another. This is different from clustering or xref:classification-algorithms:../index.adoc[classification] in general, because those types of categorization do not necessarily consider connections.
```

Comment:
The `xref` is not linked correctly. You don't need to provide the relative path - just `xref:classification-algorithms:index.adoc` will work.

Reply:
I would also use fewer pronouns.

Community algorithms are different from clustering or classification algorithms because clustering and classification do not necessarily consider connections.